### PR TITLE
Add certificates to the Oak Docker image

### LINKED ISSUE #808: Add certificate file to Docker images
After https://github.com/project-oak/oak/pull/701 we need to update the Docker image generation to include the certificate file from `./examples/certs/`.

### PR DESCRIPTION
**Pull request by ipetr0v**
This change:
- Adds certificate files to the Oak Docker image
- Generates new test keys/certificates that take into account an external experimental gcloud address for the Aggregator example

Fixes #808

# Checklist
- [x] Pull request includes prototype/experimental work that is under
      construction.

### CODE REVIEW

**Comment by ipetr0v** [comment]
cc @rbehjati

**Comment by ipetr0v** [comment]
@tiziano88 IIUC certs include information about the IP address to listen on:
```bash
$ openssl x509 -in local.pem -text -noout
...
X509v3 Basic Constraints: 
    CA:FALSE
X509v3 Key Usage: 
    Digital Signature, Non Repudiation, Key Encipherment, Data Encipherment
X509v3 Subject Alternative Name: 
    DNS:project-oak.local, IP Address:127.0.0.1, IP Address:0:0:0:0:0:0:0:1, IP Address:35.246.87.178
...

**Comment by ipetr0v** [comment]
@conradgrobler will create a separate set of keys specifically for Docker images

File: examples/certs/docker/docker.ext [hunk 1]
```diff
@@ -0,0 +1,10 @@
+authorityKeyIdentifier=keyid,issuer
+basicConstraints=CA:FALSE
+keyUsage = digitalSignature, nonRepudiation, keyEncipherment, dataEncipherment
+subjectAltName = @alt_names
+
+[alt_names]
+DNS.1 = project-oak.local
```
Comment (conradgrobler) [inline examples/certs/docker/docker.ext:7]: You can probably remove everything except for the static cloud IP address.

Comment (ipetr0v) [reply examples/certs/docker/docker.ext:7]: In this case we will not be able to test Docker images locally.

**Comment by tiziano88** [comment]
@conradgrobler @ipetr0v I believe when #696 is implemented, we will not want any kind of secret to be baked into the docker image, right? we would just provision them at startup

**Comment by ipetr0v** [comment]
Yes, I agree

**Comment by conradgrobler** [comment]
Yes.

**Pull request merged**